Source/Tests/IO/FileIdentifier.cpp

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright (c) 2022-2022 the rbfx project.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR rhs
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR rhsWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR rhs DEALINGS IN
+// THE SOFTWARE.
+//
+
+#include "../CommonUtils.h"
+
+#include <Urho3D/IO/FileIdentifier.h>
+
+TEST_CASE("FileLocator")
+{
+    FileIdentifier signleForwardSlash("file:/FileName");
```

Comment:
I am not sure file:/FileName is valid URI

Reply:
Are you sure about it? I've added this after reading documentation on the uri. It says that file:/ and file:/// are valid but file:// is not.

Reply:
Just in case I just support all three :-)

Reply:
You'll be surprised: https://en.m.wikipedia.org/wiki/File_URI_scheme

It's a big historical mess

Reply:
also I am not sure if "signle" is a correct english word ;-)